redhat/frr.spec.in

```diff
@@ -448,7 +449,8 @@ Adds GRPC support to the individual FRR daemons.
 %else
     --disable-grpc \
 %endif
-    --enable-snmp
+    --enable-snmp \
+    --enable-pcre2posix
```

Comment:
nit: there should be a `\` at the end of this line :laughing:

Reply:
Why? :) It wasn't before.

Reply:
We've tried to follow the convention of `\` lines to have an extra line at the bottom.  This way if something new comes in you just add it and don't have to modify multiple lines.  The diff just becomes what is actually changing instead of adding additional fluff to the commit

Reply:
Agree, will fix.